java/query/src/test/java/sleeper/query/lambda/SqsQueryProcessorLambdaIT.java

```diff
@@ -585,14 +601,15 @@ public void shouldPublishStatusReportsToWebSocket() throws Exception {
                         .statusReportDestinations(List.of(statusReportDestination))
                         .build())
                 .build();
-
         try {
             processQuery(query);
+            processLeafPartitionQuery();
 
             // Then
             wireMockServer.verify(1, postRequestedFor(url).withRequestBody(
-                    matchingJsonPath("$.queryId", equalTo("abc"))
-                            .and(matchingJsonPath("$.message", equalTo("completed")))
+                    matchingJsonPath("$.queryId", equalTo("abc"))));
+            wireMockServer.verify(1, postRequestedFor(url).withRequestBody(
+                    matchingJsonPath("$.message", equalTo("completed"))
                             .and(matchingJsonPath("$.recordCount", equalTo("28")))
             ));
```

Comment:
This isn't asserting that these are all the requests that were made. If we add `wireMockServer.verify(2, postRequestedFor(url));` then that would also assert that these are the only requests.

Reply:
Added to test